data/metadata.json

```diff
@@ -7,31 +7,63 @@
     }
   },
   "0x33f75ff0b830dcec": {
+    "features": {
+      "supported": ["self-custody", "mobile"]
```

Comment:
In the same vein as the comment above, what are we defining `"mobile"` as?

If this refers to FCL React Native support, no wallets have made the necessary changes to be supported in this environment yet (i.e. VIEW/MOBILE_BROWSER or VIEW/DEEPLINK implementation w/ HTTP/POST service).  This is still a change we need to get wallets to do to be compatible with React Native.

The wallets which actually "work" in mobile apps currently rely on WalletConnect which isn't shipped with FCL out of the box.

I think these issues surrounding mobile dApp support are part of a larger topic/discussion, sorry to put it here, but it's worth mentioning since it's kind of on topic.

Reply:
Let's remove the mobile feature until this is further along

Reply:
Removing mobile for now